packages/@aws-cdk/aws-eks/lib/k8s-manifest.ts

```diff
@@ -154,6 +154,8 @@ export class KubernetesManifest extends CoreConstruct {
         SkipValidation: props.skipValidation,
       },
     });
+
+    this.node.defaultChild = this.node.defaultChild?.node.defaultChild;
```

Comment:
To make this less implicit, maybe just do something like this:

```ts
const customResource = new CustomResource(this, 'Resource', ...);
this.node.defaultChild = customResource.node.defaultChild;
```

Reply:
Yes, that's much better. Thanks!